docs/products/kafka/howto/request-access-topic.md

```diff
@@ -66,9 +90,90 @@ After submitting:
 - If approved, you can view and download the credentials for authentication in
   <ConsoleLabel name="Streaming catalog"/> > **Access overview**.
 
-## View and download service user credentials
+</TabItem>
+<TabItem value="terraform" label="Terraform">
+
+Use the `aiven_governance_access` Terraform resource to request access to an Apache Kafka
+topic. The request is reviewed and approved in a GitHub pull request before access is
+granted.
+
+<div style={{ marginTop: "1.25rem" }} />
+
+### How it works
+
+1. Define the request:
+
+   - Use Terraform to define the service user, topic, and the required access control
+     lists (ACLs).
+   - Specify the `owner_user_group_id` to indicate the group responsible for approving
+     the request.
+
+     **Example Terraform configuration:**
+
+     ```hcl
+     resource "aiven_governance_access" "example" {
+      organization_id = data.aiven_organization.main.id
+      access_name     = "example-topic-access"
+      access_type     = "KAFKA"
+
+      access_data {
+        project      = data.aiven_project.main.project
+        service_name = aiven_kafka.main.service_name
+
+        acls {
+          resource_name   = "example-topic"
+          resource_type   = "Topic"
+          operation       = "Read"
+          permission_type = "ALLOW"
+          host            = "*"
+        }
+      }
+
+      owner_user_group_id = aiven_organization_user_group.example.group_id
+     }
+     ```
 
-After your request is approved, you can view and download the credentials for the
+   - Commit and push the configuration to a GitHub repository with governance approval
+     workflows enabled.
+
+1. Review and approve the request:
+
+   - The request appears as a pull request in GitHub.
+   - A GitHub Action checks the request:
+     - The requester must belong to the group defined by `owner_user_group_id`.
+     - An approval must come from another member of the same group.
+
+     :::note
+       To verify group membership, GitHub user IDs must be mapped to Aiven user IDs using
+       the
+       [`aiven_external_identity`](/docs/products/kafka/howto/terraform-governance-approvals#step-2-map-github-users-to-aiven-identities)
+       resource.
+        :::
+
+   - If the request meets all governance rules, the workflow applies the configuration
+     using `terraform apply`.
+
+1. Provision access:
+
+   - Aiven creates the service user and applies the ACLs to the specified topic.
```

Comment:
Maybe this could go in the step above or below since there isn't anything for the user to do exactly (except download the credentials, which is the next step).